LCM/dsc/engine/ConfigurationManager/LocalConfigManagerHelper.c

```diff
@@ -3175,8 +3183,9 @@ void LCM_WriteMessage_Internal_TimeTaken(
             
         if (intlstr.str)
         {
-            //// No need when OMI is not in the picture
-            // MI_Context_WriteMessage(context, channel, intlstr.str);                
+#if !defined(BUILD_OMS)
+            MI_Context_WriteMessage(context, channel, intlstr.str);                
```

Comment:
why don't we have the host(dsc_host) implement those callback methods so that the code will be identical for OMS and non-oms.

Reply:
There is no point for omsconfig to have those. No use.